features/push.yml

```
@@ -2,3 +2,39 @@ name: Push messages
 description: The Push API subscribes to and receives server-initiated messages. Subscribers receive pushed messages in the background, even after periods inactive or offline.
 spec: https://w3c.github.io/push-api/
 caniuse: push-api
+status:
+  compute_from: api.PushManager
```

For better alignment with caniuse:

```suggestion
  compute_from: api.PushMessageData
```